### ISSUE TITLE
home screen customization

### ISSUE THREAD

**Issue by Draic** [MM2-0/Kvaesitso#1079]
search first paradigm is fine but I think it is just waste to force a blank home screen. 

The only option to fill it is by disabling the clock filling it, but then the dock is no longer DOCKED to the bottom but instead somewhere top. Arbitrary locking out the option for more apps on the home screen or other ways to shortcut important stuff is also important. Why not atleast allow a full favorite bar instead of single row forced down on users? These options would barely add work, but right now the launcher feels too restrictive

**Comment by candroid-man** [comment]
Agreed, I think this needs more attention. Having only 5 apps on the homepage is extremely limiting and you should be able to add more if you wish

**Comment by Nevrai** [comment]
This hasn’t been addressed yet? That’s unfortunate.

**Comment by Fredrik108** [comment]
Totally agree, an option that enables keeping the dock at the bottom and letting the app-widget be above it would already work perfectly for me.

**Comment by uestueab** [comment]
Can't believe this is still not being addressed
